android/app/src/googleplay/java/org/fossasia/openevent/utils/map/ClusterRenderer.java

```diff
@@ -0,0 +1,39 @@
+package org.fossasia.openevent.utils.map;
+
+import android.content.Context;
+
+import com.google.android.gms.maps.GoogleMap;
+import com.google.android.gms.maps.model.Marker;
+import com.google.android.gms.maps.model.MarkerOptions;
+import com.google.maps.android.clustering.ClusterManager;
+import com.google.maps.android.clustering.view.DefaultClusterRenderer;
+
+import org.fossasia.openevent.R;
+
+/**
+ * Created by rohanagarwal94 on 4/7/17.
+ */
+public class ClusterRenderer extends DefaultClusterRenderer<MicrolocationClusterWrapper> {
+
+    private Context context;
+
+    public ClusterRenderer(Context context, GoogleMap map,
+                             ClusterManager<MicrolocationClusterWrapper> clusterManager) {
+        super(context, map, clusterManager);
+        this.context=context;
+    }
```

Comment:
It was not a question :smile: You are holding a reference in this ClusterRenderer, is it lifecycle based component or not. If we don't know the answer, we must make the context in it null when the fragment is destroyed